### PR TITLE
OCPBUGS-36146: Adding temporary warning to netobserv CLI install

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OCPBUGS-36146
https://issues.redhat.com/browse/CLOUDDST-23454
Link to docs preview:

Installing: https://78321--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/netobserv_cli/netobserv-cli-install.html
Release notes: https://78321--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/network-observability-operator-release-notes#network-observability-technology-preview-1.6_network-observability-operator-release-notes-v0
QE review:
- [X] QE has approved this change.

Additional information:

Note to peer review: A couple more approvals for CM. Going ahead with peer review as this is a slightly urgent matter.

### CODE REVIEW

**Review by jotak — APPROVED** [review]

File: snippets/network-observability-netobserv-cli-install-warning.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// Text snippet included in the following modules:
+//
+// * observability/network_observability/network-observability-operator-release-notes.adoc
+// * observability/network_observability/netobserv_cli/netobserv-cli-install.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[WARNING]
+====
+The Network Observability CLI (netobserv cli) is temporarily unavailable and is expected to resolve with link:https://issues.redhat.com/browse/OCPBUGS-36146[OCPBUGS-36146].
```
Comment (mburke5678) [inline snippets/network-observability-netobserv-cli-install-warning.adoc:14]: [Above](https://github.com/openshift/openshift-docs/pull/78321/files#diff-973c7810a7a5b24bb5b7c584b36d37727bcf80bd2b6c2ff79f050e4e661741dfR11) the parenthetical is (`oc netobserv`)
```suggestion
The Network Observability CLI (`oc netobserv`) is temporarily unavailable and is expected to resolve with link:https://issues.redhat.com/browse/OCPBUGS-36146[OCPBUGS-36146].
```

**Comment by mburke5678** [comment]
@skrthomas One nit. Otherwise LGTM

**Review by jpinsonneau — APPROVED** [review]

File: snippets/network-observability-netobserv-cli-install-warning.adoc (hunk 1 above)
Comment (skrthomas) [reply snippets/network-observability-netobserv-cli-install-warning.adoc:14]: omg, this is so stupid of me. Thanks Michael 🤦‍♀️

**Comment by ahardin-rh** [comment]
LGTM!

**Review by ahardin-rh — APPROVED** [review]
:+1:

**Comment by kalexand-rh** [comment]
LGTM

**Review by mcurry-rh — APPROVED** [review]
LGTM!

**Pull request merged**